esmvaltool/diag_scripts/monitor/multi_datasets.py

```diff
@@ -65,6 +68,14 @@
       :func:`esmvalcore.preprocessor.regrid` for this). Input data
       needs to be 2D with dimensions `time`, `latitude`/`longitude`.
 
+    Benchmarking plots
```

Comment:
I think it would be nice to to briefly explain these plot types (like it's done for the other ones). Also, this list is currently not rendered properly: https://esmvaltool--3598.org.readthedocs.build/en/3598/api/esmvaltool.diag_scripts.monitor/multi_datasets.html